CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 --
```

Comment:
oh dear this closing mark for the comment is missing the `>` 🤦

Reply:
was wondering why it was commented out 😄